.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+-   repo: https://github.com/pre-commit/mirrors-isort
+    rev: v4.3.21
+    hooks:
+    - id: isort
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.6
```

Comment:
Shouldn't this use python version 3.11 since that's the version specified in the pipfile?

Reply:
Yes, It should be 3.11, my bad totally forgot to change that.